engine/src/mac-av-player.mm

```diff
@@ -544,17 +546,16 @@ - (void)setPlayer:(AVPlayer *)player
     bool t_has_video;
     t_has_video = ( [[[[t_self -> m_player currentItem] asset] tracksWithMediaType:AVMediaTypeVideo] count] != 0);
     
-    if (!t_has_video)
-    {
-        t_self -> HandleCurrentTimeChanged();
-        return kCVReturnSuccess;
-    }
-    
-    CMTime t_output_item_time = [t_self -> m_player_item_video_output itemTimeForCVTimeStamp:*inOutputTime];
+    CMTime t_output_item_time;
```

Comment:
Init this with 0, as it will stop an potential error on some compilers as it is used later on